### DISCUSSION TITLE
NVStrapsRebar.exe doesn't open.

### DISCUSSION THREAD

**Discussion by TheTubra** [terminatorul/NvStrapsReBar#55] [General]
The executable won't open. I've tried double clicking and right-clicking as admin and absolutely nothing happens. Any ideas?

**Comment by terminatorul** [comment]
Open `cmd` and run it as Administrator first. From the open cmd window, manually type the (path and) name of the NvStrapsReBar.exe executable and run it.

If that also does not work It is possible an anti-virus is blocking it, after it notices the executable wants to write to an EFI variable. NvStrapsReBar.exe uses a non-volatile EFI variable to configure the UEFI DXE driver (NvStrapsReBar.ffs).

Run Windows Update to make use the Windows Defender anti-virus definitions are up-to-date.

If you want, you can manually build the executable from source. The main project page has a link to the build instructions.

**Reply by TheTubra** [reply]
Got it working with the cmd(admin) prompt within the location of the exe. Working just fine. Thanks so much!
